releases.yaml

```diff
@@ -9,44 +9,50 @@ latest:
   past_eol_date: false
   release_notes_url: "https://discourse.ubuntu.com/t/impish-indri-release-notes/21951"
 lts:
-  slug: FocalFossa
+  slug: JammyJellyfish
+  name: "Jammy Jellyfish"
+  short_version: "22.04"
+  full_version: "20.04"
+  release_date: "April 2022"
+  eol: "April 2027"
+  release_notes_url: "https://discourse.ubuntu.com/t/jammy-jellyfish-release-notes/24668"
+previous_lts:
   name: "Focal Fossa"
   short_version: "20.04"
   full_version: "20.04.4"
   release_date: "April 2020"
   eol: "April 2025"
-  release_notes_url: "https://wiki.ubuntu.com/FocalFossa/ReleaseNotes"
-openstack_lts:
-  slug: Ussuri
-previous_lts:
+previous_previous_lts:
   name: "Bionic Beaver"
   short_version: "18.04"
   full_version: "18.04.6"
-  release_date: "April 2018"
-  eol: "April 2023"
-previous_previous_lts:
-  name: "Xenial Xerus"
-  short_version: "16.04"
-  full_version: "16.04.7"
+openstack_lts:
+  slug: Yoga
 
 checksums:
   desktop:
+    "22.04": "update-me *ubuntu-22.04-desktop-amd64.iso"
```

Comment:
Would it be better to mark this PR as blocked/don't merge until we can get the actual checksums for these?

Reply:
Can do, I just made building some of the pages a little clearer.

Reply:
if you don't have this merged, it is really hard to make sure all the logic works on the download pages.